build/clean-locks.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Utility for cleaning package locks.
+ * Usage: node build/clean-locks.js
```

Comment:
how would you feel about changing this to "clean-package-locks"? That would communicate more accurately what this is doing - I might not know at a glace what a "lock" is nor why I might clean it.